lang/lang_en_de.txt

```diff
@@ -248,7 +248,7 @@
 
 #MSG_FSENS_NOT_RESPONDING c=20 r=4
 "ERROR: Filament sensor is not responding, please check connection."
-"FEHLER: Filament- sensor reagiert nicht, bitte Verbindung pruefen."
+"FEHLER: Filamentsensor reagiert nicht, bitte Verbindung pruefen."
```

Comment:
the Filament- sensor is meant to be written like this, because the text gets wrapped after Filament-